middleware.ts

```
@@ -1,11 +1,18 @@
 import { clerkMiddleware, createRouteMatcher } from '@clerk/nextjs/server';
 
+// Define protected routes
 const isProtectedRoute = createRouteMatcher([
```

ESLint

[error] 'isProtectedRoute' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)